ctx.go

```diff
@@ -925,6 +927,32 @@ func (c *Ctx) QueryParser(out interface{}) error {
 	return c.parseToStruct(queryTag, out, data)
 }
 
+func parseQuery(k string) string {
+	// split all the character
+	chars := strings.Split(k, "")
```

Comment:
Think this is not necessary, you can reach any letter in a string with the number

Just loop until the end of the string is reached

We have to be careful not to slow down the old functionality or to slow it down as little as possible

Would suggest the loop or a check with contains or index
And do all operations to create the new string with a bytebuffer from the pool to not produce allocations.

Reply:
Yeah true, I will change to byteBuffer from pool, this is my first PR so didn't know there are benchmark tests to follow :)